diff -rup tuigreet-0.9.1.orig/Cargo.toml tuigreet-0.9.1/Cargo.toml
--- tuigreet-0.9.1.orig/Cargo.toml	2024-05-30 14:51:39.000000000 -0700
+++ tuigreet-0.9.1/Cargo.toml	2024-06-02 11:36:39.985385100 -0700
@@ -26,7 +26,7 @@ crossterm = { version = "^0.27", feature
   "crossterm",
 ] }
 rust-embed = "^8.0"
-rust-ini = "^0.21"
+rust-ini = ">=0.18,<0.22"
 smart-default = "^0.7"
 textwrap = "^0.16"
 tokio = { version = "^1.2", default-features = false, features = [
