--- yaml-rust2-0.10.1/Cargo.toml	1970-01-01T00:00:01+00:00
+++ yaml-rust2-0.10.1/Cargo.toml	2025-05-01T18:20:17.457795+00:00
@@ -79,11 +79,6 @@
 name = "test_round_trip"
 path = "tests/test_round_trip.rs"
 
-[[test]]
-name = "yaml-test-suite"
-path = "tests/yaml-test-suite.rs"
-harness = false
-
 [dependencies.arraydeque]
 version = "0.5.1"
 
@@ -92,10 +87,7 @@
 optional = true
 
 [dependencies.hashlink]
-version = "0.10"
-
-[dev-dependencies.libtest-mimic]
-version = "0.4.0"
+version = ">=0.9,<0.11"
 
 [dev-dependencies.quickcheck]
 version = "1.0"
