diff -ru protox-0.9.1/tests/compare.rs protox-0.9.1-new/tests/compare.rs
--- protox-0.9.1/tests/compare.rs	2006-07-24 03:21:28.000000000 +0200
+++ protox-0.9.1-new/tests/compare.rs	2026-03-02 18:49:16.853229683 +0100
@@ -63,6 +63,7 @@
         .arg(google_src_dir())
         .arg("--include_imports")
         .arg("--include_source_info")
+        .arg("--experimental_allow_proto3_optional")
         .arg(format!("--descriptor_set_out={}", result.display()))
         .args(files)
         .stderr(Stdio::piped())
