Subject: [PATCH] Update postmark to 0.11
---
Index: crates/atuin-server/Cargo.toml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/crates/atuin-server/Cargo.toml b/crates/atuin-server/Cargo.toml
--- a/crates/atuin-server/Cargo.toml	(revision 57eecfd742a52982f5aec5e45e00e3b29cebc6a0)
+++ b/crates/atuin-server/Cargo.toml	(revision 8974b69c0c0d8a80df60e48a44f999fbec61c494)
@@ -34,4 +34,4 @@
 semver = { workspace = true }
 metrics-exporter-prometheus = "0.16"
 metrics = "0.24"
-postmark = {version= "0.10.2", features=["reqwest", "reqwest-rustls-tls"]}
+postmark = {version= "0.11", features=["reqwest", "reqwest-rustls-tls"]}
Index: crates/atuin-server/src/handlers/user.rs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/crates/atuin-server/src/handlers/user.rs b/crates/atuin-server/src/handlers/user.rs
--- a/crates/atuin-server/src/handlers/user.rs	(revision 57eecfd742a52982f5aec5e45e00e3b29cebc6a0)
+++ b/crates/atuin-server/src/handlers/user.rs	(revision 8974b69c0c0d8a80df60e48a44f999fbec61c494)
@@ -221,7 +221,7 @@
 
     let client = PostmarkClient::builder()
         .base_url("https://api.postmarkapp.com/")
-        .token(postmark_token)
+        .server_token(postmark_token)
         .build();
 
     let req = postmark::api::email::SendEmailRequest::builder()
